frontend/src/pages/edItVault/vaultDeleteSettings/DeleteVaultPage.tsx

```diff
@@ -70,7 +70,7 @@ const DeleteVaultPage = () => {
       if (vaults.length > 0) {
         navigate(makeAppPath('vaults'));
       } else {
-        navigate(makeAppPath('setupVault', {}));
+        navigate(appPaths.root);
```

Comment:
FYI, You can also now use `useAppNavigate`, it would require you to pass parameters if necessary.